easybuild/easyconfigs/l/launcher/launcher-3.7-foss-2020a.eb

```diff
@@ -0,0 +1,51 @@
+# Autor:  Felipe Franco <felipe.franco@hpcnow.com>
+#
+easyblock = 'Tarball'
+
+name = 'launcher'
+version = '3.7'
+
+homepage = 'https://github.com/TACC/launcher'
+description = """
+Launcher is a utility for performing simple, data parallel, 
+high throughput computing (HTC) workflows on clusters, 
+massively parallel processor (MPP) systems, workgroups of computers, 
+and personal machines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/tacc',
+        'repo_name': 'launcher',
+        'tag': 'v3.7',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+
+checksums = ['7a3cfc16e4882d9fa08093b1035b1f5e8733477246ca1088cfdc27fc30bf9b90']
+
+dependencies = [
+    ('hwloc', '2.2.0', '', ('GCCcore', '9.3.0')),
+]
+
+
+sanity_check_paths = {
+    'files': ['launcher', 'tests/quicktest'],
+    'dirs': ['tests'],
+}
```

Comment:
Please also consider adding a `sanity_check_command`, which could be a simple usage message with `--help`, for example.

Reply:
Launcher creates bash scripts that seem to not be supported by sanity_check_commands parameter. Don't you think that it is the function of the parameter test in the recipe? And, then, no need of using sanity_check_commands

Reply:
@verdurin the recipe already runs two sanity tests.
Unfortunately, there is no command to be executed to test the installation other than running again the same tests. https://github.com/TACC/launcher/tree/master/tests